internal/veracode-api-wrapper/updatemitigationinfo/README.md

```diff
@@ -0,0 +1,48 @@
+**Veracode API Wrapper - Update Mitigation Info**
+
+Calls the Veracode api to update the mitigation comment of a build id. This is effectively used in updating the comment 
+in veracode which is maintained in CSV files.
+
+**Prerequisite**
+
+Prerequesties specific to Veracode :  Veracode Subcription , User role with "Mitigation" and "Mitigation and Comments"
```

Comment:
Shall we make this a h3 (###) and use a bulleted list. Same for the next point. Please also correct the typo in the word `specif`.